examples/oracle-example/src/lib.rs

```diff
@@ -18,6 +22,7 @@ pub struct UserConfig {}
 #[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
 pub struct AuxData {}
 
+pub const ORACLE_DATA: [&str; 1] = ["block_number_entropy"];
```

Comment:
```suggestion
pub const ORACLE_DATA: [&str; 1] = ["block_number_entropy"];

```

Since this is an example there should also be a comment here explaining what this is and how to use it.